components/list/list-item-generic-layout.js

```diff
@@ -68,33 +68,34 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 					[content-end actions-start] minmax(0, min-content)
 					[end actions-end];
 				grid-template-rows: [main-start] [main-end nested-start] [nested-end];
-				position: relative;
 			}
 
-			::slotted([slot="nested"]) {
-				grid-column: content-start / end;
-				grid-row: nested-start / nested-end;
-			}
 			:host([align-nested="control"]) ::slotted([slot="nested"]) {
 				grid-column: control-start / end;
 			}
 			:host(.d2l-dragging-over) ::slotted([slot="nested"]) {
-				order: 7; /* must be greater than item's drop-target to allow dropping onto items within nested list  */
+				z-index: 2; /* webkit dom order fix */
 			}
 
 			::slotted([slot="drop-target"]) {
-				height: 100%;
-				order: 6;
-				position: absolute;
-				top: 0;
-				width: 100%;
+				grid-column: 1 / -1;
```

Comment:
Is it possible to use the grid line names?

Reply:
I wanted to avoid names for this one since it should likely fill the row regardless of future column changes.

Reply:
Would that correspond to the `start` and `end` grid names that are defined above? Not a big deal though.

Reply:
Hah, yep. I forgot they were named twice like the others. That's totally reasonable. I'll switch to those.